script/Common/00_00.txt

```diff
@@ -712,7 +712,7 @@ From the moment I realized that, it was as though my ears had been frozen shut.
 [sha:10521369a69f688adcdec08cef06e4af9e26df2a]{
 -- Page 50, Offset 243.
 -- 胸の傷だけがサイレンのように響いている。
-Only the wound in my chest reverberates like a siren.
+Only the wound on my chest reverberates like a siren.
```

Comment:
I kinda like in because it's _deeper_

Reply:
it sounds really really wrong to me, feel kinda strongly about keeping "on"